plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -219,14 +218,10 @@ func (l *persistentVolumeLabel) getEBSVolumes() (aws.Volumes, error) {
 		if err != nil || cloudProvider == nil {
 			return nil, err
 		}
-		awsCloudProvider, ok := cloudProvider.(*aws.Cloud)
-		if !ok {
-			// GetCloudProvider has gone very wrong
-			return nil, fmt.Errorf("error retrieving AWS cloud provider")
-		}
-		l.ebsVolumes = awsCloudProvider
+
+		l.awsPVLabeler = cloudProvider.(cloudprovider.PVLabeler)
```

Comment:
this will panic if the interface is not implemented. seems like we want an error, as before